backend/services/api.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+
+def get_pokemon_list():
+    url = "https://pokeapi.co/api/v2/pokemon/?limit=50"
+    response = requests.get(url)
+    response = response.json()
+    return response["results"]
```

Comment:
One more suggestion and again it isn't wrong the way you did, feel free to choose what you prefer :)

```suggestion
def get_pokemon_list():
    response = requests.get(f"{POKEAPI_ROOT_URL}?limit=50")
    if not response.ok:
        return None   
    return response.json()["results"]

```